src/components/SellerPage/SellerPage.tsx

```diff
@@ -52,6 +50,13 @@ const SellerPage: React.FC<{}> = () => {
               imageSrc={sampleMerchant.ownerInfo.imageSrc}
               sellerName={seller.name}
               progressBarColor={seller.progress_bar_color}
+              extraInfo={{
```

Comment:
Nice have you tested this with the backend?

Reply:
yep, works with extra info columns